commisery/cli/__init__.py

```diff
@@ -175,10 +175,12 @@ def _read_commit_message():
 
         answers = inquirer.prompt(questions)
 
-        answers["scope"] = answers.get("scope") if answers.get("scope") else None
+        answers["scope"] = answers.get("scope") if answers.get("scope", None) else None
         answers["type"] = answers.get("type").split(":")[0]
-        answers["breaking_change"] = "!" if answers.get("breaking_change") else ""
-        answers["body"] = (os.linesep + answers.get("body")).split(os.linesep)
+        answers["breaking_change"] = "!" if answers.get("breaking_change", "No") == "Yes" else ""
```

Comment:
Need to validate the actual response (either `Yes` or `No`)...

Reply:
Does this mean people need to type `Yes` with exactly that capitalization?

(Hint: If so, then that's a terrible UI for obtaining boolean values)